src/status_im/subs.cljs

```diff
@@ -2953,6 +3002,27 @@
  (fn [multiaccount]
    (pos? (count (get multiaccount :images)))))
 
+(re-frame/reg-sub
+ :profile/multiaccount
```

Comment:
nit: should the variable name be `multi-account`?

Reply:
i wish name it multi-account too...but the old code called it multiaccount, so i just want keep the same :)